npc/re/merchants/prt_market.txt

```diff
@@ -0,0 +1,206 @@
+//===== rAthena Script ======================================= 
+//= Prontera Market Street
+//===== Changelogs: ==========================================
+//= 1.0 Initial version from replays. [Haydrich]
+//============================================================
+
+prt_mk,161,171,0	script	Guide#mk_prt5	8W_SOLDIER,{
+	cutin "prt_soldier.bmp", 2;
+	.@npc_name$ = strnpcinfo(0);
+	.@menu$[0] = "Move to the East Market Street";
+	.@menu$[1] = "Move to the South Market Street";
+	.@menu$[2] = "Move to the West Market Street";
+	.@menu$[3] = "Move to the North Market Street";
+	.@menu$[4] = "~ Move to the Center of Prontera";
+	.@menu$[5] = "Cancel";
+	setarray .direction$[0], "to the East", "to the South", "to the West", "to the North";
+	setarray .warp_x, 285, 161, 38, 161;
+	setarray .warp_y, 162, 39, 162, 287;
```

Comment:
There is no point to use `.var` here, please use `.@var`.

Write these default data in a `else` instead, like
```
else if (.@npc_name$ == "Guide#mk_prt4") {
...
}
else {
```